### PR TITLE
Add scheduler with monthly report

### PR DESCRIPTION
**Pull request by justmendes**

### CODE REVIEW

File: config/prod.exs [hunk 1]
```diff
@@ -6,3 +6,10 @@ config :corker, Corker.Repo,
   ssl: true,
   url: System.get_env("DATABASE_URL"),
   pool_size: String.to_integer(System.get_env("POOL_SIZE") || "10")
+
+config :corker, Corker.Scheduler,
+  jobs: [
+    {"12 0 1 * *", {Corker.Jobs.Report, :send, []}}
```
Comment (jfranciscosousa) [inline config/prod.exs]: Why 12? `0 0 1 * *` works at `00:00` first day of month always.

Comment (justmendes) [reply config/prod.exs]: At `00:00` nobody is on Slack, `12:00` seemed like a nice time to have people see this.

Comment (jfranciscosousa) [reply config/prod.exs]: Then you should do `0 12 1 * *`, if this is regular crontab syntax

Comment (jfranciscosousa) [reply config/prod.exs]: https://crontab.guru/#0_12_1_*_*

Comment (justmendes) [reply config/prod.exs]: Yep, you're right, that's what you get for not copy/pasting from crontab guru

**Pull request merged**